lib/flags.js

```diff
@@ -23,8 +23,8 @@ module.exports = {
     const result = {};
     const https = parseGroup(argv, 'https');
     const open = parseGroup(argv, 'open');
-    const { devWare: devMiddleware } = argv;
-    let { hotClient } = argv;
+    const devMiddleware = argv.devWare || {};
+    const hotClient = argv.hotClient || {};
```

Comment:
what do you think about:
```
const { devMiddleware = {}, hotClient = {} } = argv;
```

Reply:
that would result in a `TypeError: Cannot match against 'undefined' or 'null'.` on Node 6.11.0